src/blog/_posts/2023-08-01-introduction-to-cadence-signals.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Introduction to Cadence Signals
+
+date: 2023-08-06
+author: Chris Qin
+authorlink: https://www.linkedin.com/in/chrisqin0610/
+---
+
+In this blog, we will have a brief overview of Cadence signals, a critical feature that provides a way to directly send data to a running workflow. 
```

Comment:
'In this blog we will give you a brief overview.....' 

It would be good to highlight Cadence Signals in quotes, italics or bold as you are defining what it is.